component/libfuse/libfuse2_handler.go

```diff
@@ -642,6 +642,7 @@ func (cf *CgofuseFS) Open(path string, flags int) (int, uint64) {
 
 // Read reads data from a file into the buffer with the given offset.
 func (cf *CgofuseFS) Read(path string, buff []byte, ofst int64, fh uint64) int {
+	log.Debug("Libfuse::Read : reading path %s, handle: %d", path, fh)
```

Comment:
Log output was intentionally skipped in `Read` and `Write` to avoid creating log noise and the performance costs of a huge number of calls.
You could just drop this log call, or maybe the most readable thing would be to comment it out and add a comment explaining why we don't have log output here.